### ISSUE TITLE
Idea: fail benchmarks if miner CPU utilisation is too high

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#1235]
Recently there have been issues with miners hogging all the CPU. https://github.com/RainbowMiner/RainbowMiner/issues/1234 https://github.com/RainbowMiner/RainbowMiner/issues/1228

I have an idea to fix this for ALL miners. If RBM monitors the CPU utilisation of the mining process during benchmarking and then fails it if it is too high.

**Comment by samdave123** [comment]
Another related idea: RBM records the CPU usage during a benchmark and display it on the benchmarks stats. At least then I can see the miners with issues and disable them myself.

**Comment by RainbowMiner** [comment]
Good ideas. I'll think about a solution.

**Comment by samdave123** [comment]
Thank you. Weirdly, NBMiner is doing this now but it's been a very stable miner for a long time until recently.

**Comment by RainbowMiner** [comment]
Ah, I have an idea. It could be, that one of your GPUs has crashed. That would cause overly high CPU load for most of the miners, because they fail to access the hardware properly.
Did you already try to switch your rig off and back on?

**Comment by samdave123** [comment]
Yes I did some troubleshooting on this. I rebooted the rig and then set core and memory to +0 and it was still hogging up my CPU.
